content/en/news/releases/1.4.x/announcing-1.4.7/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.4.7
+linktitle: 1.4.7
+subtitle: Patch Release
+description: Istio 1.4.7 patch release.
+publishdate: 2020-03-03
+release: 1.4.7
+aliases:
+    - /news/announcing-1.4.7
+---
+
+This release note describes what’s different between Istio 1.4.6 and Istio 1.4.7.
+
+{{< relnote >}}
+
+## Security Update
+
+TODO
+
+## Changes
+
+* **Fixed** an issue causing protocol detection to break HTTP2 traffic to gateways ([Issue 21230](https://github.com/istio/istio/issues/21230)).
```

Comment:
Is this fixed in 1.4.7? Issue says it was closed February 19 and 1.4.6 came out March 3.

Reply:
Nevermind. Go down the rabbit hole and it merged around the same time. 

https://github.com/istio/istio/pull/21727